src/applications/personalization/profile/actions/index.js

```diff
@@ -15,6 +17,24 @@ export const FETCH_MILITARY_INFORMATION_FAILED =
 export const FETCH_ADDRESS_CONSTANTS_SUCCESS =
   'FETCH_ADDRESS_CONSTANTS_SUCCESS';
 
+const captureMilitaryInfoErrorResponse = ({ error, apiEventName }) => {
+  const [firstError = {}] = error.errors ?? [];
```

Comment:
FUTURE WORK: This logic seems to be getting repeated a few times, we should look at refactoring this logic, no rush, but just to say it outloud

Reply:
agreed